cumulusci/tasks/metadata/managed_src.py

```diff
@@ -7,6 +7,9 @@
 
 
 class CreateManagedSrc(BaseTask):
+    task_docs = """The //cumulusci-managed string is to guard the @deprecated
+annotation which is not legal to deploy in an unmanaged context."""
```

Comment:
How about:

    task_docs = """Apex classes which use the @deprecated annotation can comment it out using //cumulusci-managed so that it can be deployed as part of unmanaged metadata, where this annotation is not allowed. This task is for use when deploying to a packaging org to remove the comment so that the annotation takes effect."""